tests/tests-8/user_volume/test.sh

```diff
@@ -72,6 +79,6 @@ if ! grep -E -q 'One Dell Way Round Rock, Texas 78682' <<<"$data"; then
 fi
 container_cleanup "$container_name"
 
-rm -fr myconf myvarsolr mylogs
+rm -fr $myconf $myvarsolr $mylogs $configsets
```

Comment:
spellcheck wants those to be quoted. If you run:

```
shellcheck -e SC1090,SC2001 $(find . -name '*.sh')                                                                     
```

there are various other recommendations for the changes too. It's a bit of a faff, but worth it.

Reply:
Done